docker-compose.yml

```diff
@@ -52,11 +52,30 @@ services:
       - CRC_ENDPOINT_ERROR=http://error
       - SPLUNK_URL=https://dps-splunk:8088
       - SPLUNK_TOKEN=${SPLUNK_TOKEN}
+      - SWAGGER_UI_ENABLED=true
     ports:
       - "8081"
     networks:
       - dps-net
 
+  #############################################################################################
+  ###                            FIGARO validation service                                  ###
+  #############################################################################################
+  fig-validation-service:
+    build:
+      context: ./src
+      args:
+        - spring_profiles_active=cloud
+        - MVN_PROFILES=figvalidationservice,splunk
+        - DPS_SERVICE_NAME=figvalidationservice
+    environment: 
+      - SWAGGER_UI_ENABLED=true
+    ports:
+      - "8084:8081"
+    networks:
```

Comment:
Curious why port 8084 is defined here. What is using 8084?

Reply:
Same. So many magic numbers here, I need education.

Reply:
yes it's random. I'm thinking going 8080 is nginx and then up 1 for all services